cmd/kubeadm/app/master/manifests.go

```diff
@@ -233,7 +233,7 @@ func getComponentCommand(component string, s *kubeadmapi.MasterConfiguration) (c
 			"--tls-cert-file=" + pkiDir + "/apiserver.pem",
 			"--tls-private-key-file=" + pkiDir + "/apiserver-key.pem",
 			"--token-auth-file=" + pkiDir + "/tokens.csv",
-			"--secure-port=443",
+			fmt.Sprintf("--secure-port=%d", s.API.BindPort),
```

Comment:
I didn't introduce `s` here, it's been around for a while. Nevertheless, I'm happy rename it.